galaxy_ng/app/tasks/promotion.py

```diff
@@ -1,13 +1,37 @@
 from pulpcore.plugin.tasking import add_and_remove, dispatch
 from pulp_ansible.app.models import CollectionVersionSignature
+from pulpcore.plugin.models import TaskGroup
+
+
+def call_add_content_task(collection_version, repo):
+    """Dispatches the add content task
+
+    This is a wrapper to copy_content task
+    """
+
+    signatures_pks = CollectionVersionSignature.objects.filter(
+        signed_collection=collection_version.pk,
+        pk__in=repo.content.values_list("pk", flat=True)
+    ).values_list("pk", flat=True)
+
+    task_group = TaskGroup.current()
+
+    return dispatch(
```

Comment:
After this `dispatch` you should call `task_group.finish()` to change the 'dispatching tasks' state on the group report to finished.